tests/hypertext/HTMLLintBearTest.py

```diff
@@ -27,3 +27,10 @@
     invalid_files=(test_file,),
     settings={'htmllint_ignore': 'quotation'},
     tempfile_kwargs={'suffix': '.html'})
+
+HTMLLintBearDisableCheckerTest = verify_local_bear(
+    HTMLLintBear,
+    valid_files=(test_file,),
+    invalid_files=(),
```

Comment:
@namanyadav12 could you please view HTMLLintBearTest.py in its entirety and not just the changes I made